src/test/java/org/janelia/saalfeldlab/n5/AbstractN5Test.java

```diff
@@ -708,7 +716,10 @@ public void testListAttributes() {
 			Assert.assertTrue(attributesMap.get("attr5") == long[].class);
 			Assert.assertTrue(attributesMap.get("attr6") == long.class);
 			Assert.assertTrue(attributesMap.get("attr7") == double[].class);
-			Assert.assertTrue(attributesMap.get("attr8") == Object[].class);
```

Comment:
Why should a String be accepted?

Reply:
Currently in n5-hdf5 `listAttributes` will always return a `String`. Parsing the JSON to an Object[] is deferred until `getAttribute` or `getAttributes`:

* `listAttributes`: https://github.com/saalfeldlab/n5-hdf5/blob/632cfb8b8b0b83938b58c35b0d18d66532117922/src/main/java/org/janelia/saalfeldlab/n5/hdf5/N5HDF5Reader.java#L794-L816
* `getAttribute`: https://github.com/saalfeldlab/n5-hdf5/blob/632cfb8b8b0b83938b58c35b0d18d66532117922/src/main/java/org/janelia/saalfeldlab/n5/hdf5/N5HDF5Reader.java#L392

Either it's acceptable for `listAttributes` to return a `String` in the case of JSON or this is a bug in n5-hdf5. If it's the latter, then I would be interested in fixing that.